prov/efa/src/rxr/rxr_init.c

```diff
@@ -381,6 +381,13 @@ static int rxr_info_to_rxr(uint32_t version, const struct fi_info *core_info,
 		 * which means FI_MR_HMEM implies FI_MR_LOCAL for cuda buffer
 		 */
 		if (hints->caps & FI_HMEM) {
+			if (hints->domain_attr &&
+			    !(hints->domain_attr->mr_mode & FI_MR_HMEM)) {
+				FI_INFO(&rxr_prov, FI_LOG_CORE,
```

Comment:
FI_WARN?

Reply:
I was following the other validate prints in the getinfo path that use FI_INFO, but I can change this if you want.

Reply:
If other validation is also using FI_INFO then no need to change.